File: back/src/main/java/com/myweather/api/models/helpers/SessionCredentials.java
package com.myweather.api.models.helpers;

/**
 * Created by javierfz on 7/13/17.
 */
public class SessionCredentials {

   private String email;
   private String password;

   public String getEmail() {
      return email;
   }

   public String getPassword() {
      return password;
   }

   public void setEmail(String email) {
      this.email = email;
   }

   public void setPassword(String password) {
      this.password = password;
   }
}

File: front-reactive/src/app/login/models/sessionCredentials.ts
export class SessionCredentials {

  private _email: string;
  private _password: string;

  set email(value: string) {
    this._email = value;
  }
  set password(value: string) {
    this._password = value;
  }

  get email(): string {
    return this._email;
  }

  get password(): string {
    return this._password;
  }


}
